packages/bits/src/lib/select-v2/select/select-v2.component.spec.ts

```diff
@@ -460,6 +460,14 @@ describe("components >", () => {
                     component.toggleDropdown();
                     expect(component["optionKeyControlService"].getActiveItemIndex()).toEqual(0);
                 });
+                it("should select the null option if value is null", () => {
+                    const nullOption = TestBed.createComponent(SelectV2OptionComponent).componentInstance
+                    component.selectedOptions = [nullOption];
```

Comment:
It may improve the test to add three options. Put the null one in the middle and check that the middle one is selected. This is because the first one gets selected in either case, so it could be a false positive.